src/react/components/Spinner.js

```diff
@@ -1,14 +1,31 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const Spinner = (props) => (
-  <div className="cc-spinner" aria-live="polite" aria-busy={true}>
-    {props.children && <p className="cc-spinner-message">{props.children}</p>}
-  </div>
-);
-
-Spinner.propTypes = {
-  children: PropTypes.node
+import '../stylesheets/widgets.scss';
+import '../stylesheets/spinners.scss';
+
+const propTypes = {
+  isLoadingMessage: PropTypes.string
+};
+
+const renderMessage = (isLoadingMessage) => {
+  if (isLoadingMessage) {
+    return (
+      <p className="cc-react-spinner-message">{isLoadingMessage}</p>
+    );
+  } else {
+    return null;
+  }
+};
+
+const Spinner = (props) => {
+  return (
+    <div className='cc-react-widget--padding'>
+      <div aria-live="polite" className="cc-react-spinner"></div>
+      {renderMessage(props.isLoadingMessage)}
```

Comment:
Is there a reason to extract this to another function rather than:

`{props.isLoadingMessage && <p className="cc-react-spinner-message">{props.isLoadingMessage}</p>}`

Reply:
Nope not at all, I think just stylistic differences.  I don't feel too strongly about it, I can amend if you think it's worth it!